packages/layer-composer/src/generators/vessel-events/vessel-events.ts

```diff
@@ -24,14 +24,18 @@ interface VesselsEventsSource extends GeoJSONSourceRaw {
 
 export type GlobalVesselEventsGeneratorConfig = MergedGeneratorConfig<VesselEventsGeneratorConfig>
 
-const POINTS_TO_SEGMENTS_ZOOM_LEVEL_SWITCH = 9
```

Comment:
and where is this default value to keep working the other projects?

Reply:
With @j8seangel we decided to change the default behavior to not switch to segments view here in the `vessel-events.ts` but he already modified the dataview of the vessel events to have this value set to 9 in both `dev` and `pro` so the current behavior is not changed in `fishing-map`:
```
        "config": {
            "type": "TRACK",
            "color": "#F95E5E",
            "showIcons": false,
            "showAuthorizationStatus": false,
            "pointsToSegmentsSwitchLevel": 9
        },
```